block-node/stream-publisher/src/main/java/org/hiero/block/node/stream/publisher/StreamPublisherPlugin.java

```diff
@@ -75,10 +76,15 @@ public Pipeline<? super Bytes> open(
             @NonNull final Pipeline<? super Bytes> replies) {
         final BlockStreamPublishServiceMethod blockStreamPublisherServiceMethod =
                 (BlockStreamPublishServiceMethod) method;
+
+        final int maxMessageSize =
```

Comment:
Why the subtraction?
Is it to ensure there's some wiggle room?
Would this impact the value even if was set to the min value?
Also is this required to be replicated anywhere a pipeline is being set up for the publish endpoint?